interface/patient_file/report/patient_report.php

```diff
@@ -386,6 +387,11 @@ function show_date_fun(){
                                                 " value='" . $result{"encounter"} . "'" .
                                                 " class='encounter_form' ".
                                                 ">" . xl_form_title($result{"form_name"}) . "<br>\n");
+		*/
+		 echo "<input type='checkbox' name='".$result['formdir']."_".$result['form_id']."'".
```

Comment:
I have this same code running on another server and the only difference between the two is one is on PHP 7.0.4 and the other is on PHP 5.6.16. The code that is on the php 5 does not have this issue.  However I can say that the code on the php 5, we down graded it from php 7 because we were having more problems than I could fix fast enough. 
The multiple issues when away after the down grade to php 5.x.x.